integration_tests/test_search.py

```diff
@@ -323,34 +400,116 @@ class SpecificAuthor(FirstMatch):
     Most of the results must also be by that author.
     """
 
-    def __init__(self, author, accept_book_about_author=False, threshold=0):
+    def __init__(self, author, accept_title=None, threshold=0):
```

Comment:
accept_title lets me customize what kind of title I'll accept as a book about the author. It doesn't have to be that the title of the biography is literally the author's full name.